apps/projects/app/components/Panel/PanelComponents.js

```diff
@@ -40,14 +42,111 @@ export const IssueTitle = ({ issue }) => (
 )
 IssueTitle.propTypes = issueShape
 
-export const IssueText = styled.div`
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
+export const Avatar = ({ user }) => {
+  const theme = useTheme()
+
+  return (
+    <Link
+      href={user.url}
+      target="_blank"
+      style={{ textDecoration: 'none', color: `${theme.link}`, marginRight: 6 }}
+    >
+      <img
+        alt=""
+        src={user.avatarUrl}
+        css="width: 40px; height: 40px; margin-right: 10px; border-radius: 50%;"
+      />
+    </Link>
+  )
+}
+Avatar.propTypes = {
+  user: PropTypes.shape({
+    url: PropTypes.string.isRequired,
```

Comment:
I think we only use `avatarUrl`

Reply:
`url` is for the user. `avatarUrl` is for user's avatar pic.